tests/airportDetailsRetriever.test.js

```diff
@@ -0,0 +1,44 @@
+const airport = require('../src/airportDetailsRetriever');
```

Comment:
Better to mock `const airportDetails = require('.../data/airports.json');` as test shouldn't be dependent on values in the .json file, what if DEL is removed permanently from file or TWI gets added to file. The tests would fail, purpose here is to test the logic of airport.getDetails